indico/modules/events/timetable/operations.py

```diff
@@ -42,6 +43,15 @@ def _get_object_info(entry):
     return object_type, object_title
 
 
+def create_break_entry(event, data):
+    break_ = Break()
+    entry_data = {'break_': break_,
```

Comment:
you can always pass `'object': ..` no matter which type (in the end it sets the correct relationship attribute in object.setter)